app/renderer/src/containers/AppContainer.js

```diff
@@ -29,19 +39,42 @@ const AppContainer = class extends Component {
     }
   }
 
+  updateQuery = q => {
+    this.setState({ q });
```

Comment:
I'm sure you know this better than I do, but at which stage do we run a `debounce`?

Reply:
This is handled on the main process You're right though the client should debounce the query so we're not passing too many msgs to the main process.